packages/components/pie-breadcrumb/src/index.ts

```diff
@@ -13,19 +24,93 @@ export * from './defs';
  * @tagname pie-breadcrumb
  */
 export class PieBreadcrumb extends RtlMixin(PieElement) implements BreadcrumbProps {
+    @property({ type: Array })
+    public items: BreadcrumbProps['items'] = [];
+
+    /**
+     * Renders a separator icon between breadcrumb items.
+     * The icon direction depends on the RTL (Right-to-Left) setting.
+     *
+     * @returns {TemplateResult} HTML template for the separator icon.
+     *
+     * @private
+     */
+    private renderSeparator () {
+        return html`
+            <li role="presentation" aria-hidden="true" class="c-breadcrumb-separator">
+                ${this.isRTL ? html`<icon-chevron-left></icon-chevron-left>` : html`<icon-chevron-right></icon-chevron-right>`}
+            </li>
+        `;
+    }
+
+    /**
+     * Renders a navigation link for a breadcrumb item.
+     *
+     * @param {BreadcrumbItem} item - The breadcrumb item containing label and URL.
+     *
+     * @private
+     */
+    private renderNavigationLink (item: BreadcrumbItem) {
+        return html`
+            <pie-link isStandalone="true" underline="reversed" isBold="true" href="${item.href}">
+                ${item.label}
+            </pie-link>
```

Comment:
I believe we don't have to pass true as a string to boolean values so we can simply do;

```suggestion
            <pie-link isStandalone underline="reversed" isBold href="${item.href}">
                ${item.label}
            </pie-link>
```

Reply:
Fixed here: https://github.com/justeattakeaway/pie/pull/2314/commits/38c0f38037bfeb2f6b69448e8f2bae9e7cf415e7